src/genie/libs/parser/hp_comware/tests/DisplayInterfaces/cli/equal/golden_output2_expected.py

```diff
@@ -0,0 +1,18 @@
+expected_output = {
+    "Vlan-interface805": {
+        "description": "==VLAN INTERFACE ==",
+        "enabled": True,
+        "frame_type": "PKTFMT_ETHNT_2",
```

Comment:
below are always same between `IP` and `IPv6`?
```
IP Packet Frame Type: PKTFMT_ETHNT_2,  Hardware Address: cc3e-5f69-abcd
IPv6 Packet Frame Type: PKTFMT_ETHNT_2,  Hardware Address: cc3e-5f69-abcd

Reply:
I think the user can make it different, but I can't figure why someone would want to do that.
Anyway I could add keys for both , what do you think?

Reply:
Yeah, me too. but as long as we see the output, better to collect separately. otherwise, someone will ask which one is which one or like where is ipv6 in the future...

Reply:
Fixed in 9e1932b

Reply:
confirmed the change